### PR TITLE
[RHPAM-1566] Not able to open task form from Task Inbox on business central 

### PR DESCRIPTION
**Pull request by elguardian**
instance variable is null causing code to break when it is marshalled

(cherry picked from (#2154)

### CODE REVIEW

**Comment by elguardian** [comment]
ok to test

File: kie-wb-common-forms/kie-wb-common-dynamic-forms/kie-wb-common-dynamic-forms-backend/src/test/java/org/kie/workbench/common/forms/dynamic/backend/server/context/generation/dynamic/impl/EmptyMultipleSubformBackendFormRenderingContextManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
```
[the remark this answers is not in the document]
Comment (elguardian) [reply kie-wb-common-forms/kie-wb-common-dynamic-forms/kie-wb-common-dynamic-forms-backend/src/test/java/org/kie/workbench/common/forms/dynamic/backend/server/context/generation/dynamic/impl/EmptyMultipleSubformBackendFormRenderingContextManagerTest.java]: i will change that

**Comment by elguardian** [comment]
done @Rikkola

**Comment by Rikkola** [comment]
@elguardian do you have a member from the QA that should take a look at this or should we find one?

**Comment by Rikkola** [comment]
@kkufova Looks like Klara's name was on the ticket. Hi @kkufova o/

**Comment by Rikkola** [comment]
Whoap. Wrong RHPAM that I accidentally looked at. Hi anyway @kkufova :)

**Comment by kkufova** [comment]
Hello! :-) Do you still need my help? I guess I can check the fix.

**Comment by Rikkola** [comment]
@elguardian It would also be good if you made a new ticket into DROOLS for example with the same title as the RHPAM ticket and used it in the git commit comment. Then link the two tickets together.

**Comment by Rikkola** [comment]
@kkufova If you can it would be nice. The ticket has nobody assigned to it.

**Comment by kkufova** [comment]
@Rikkola, okay, no problem, I'll check today.

**Comment by mareknovotny** [comment]
Jenkins retest this please

**Pull request merged**